src/main/java/com/icthh/xm/uaa/domain/properties/TenantProperties.java

```diff
@@ -42,9 +42,17 @@
     @JsonProperty("security")
     private Security security;
 
+    public Security getSecurity() {
```

Comment:
can we just use `private Security security = new Security();` instead of defining the getter?